src/resources/js/features/profile/Index.vue

```diff
@@ -66,7 +67,7 @@ export default {
   },
   computed: {
     canShowDetailView: function () {
-      return this.viewName === "detail";
+      return this.viewName === "detail"  && this.user;
```

Comment:
[imo]
一つスペース多い気がする 👁️ 

```suggestion
      return this.viewName === "detail" && this.user;
```